.github/workflows/build_data.yaml

```diff
@@ -0,0 +1,26 @@
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+name: build_data
+
+jobs:
+  build_data:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v1
+
+      - name: Make Glosario
+        run: Rscript utils/make-glossary.R
+
+      - name: Push the new file to the repo
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
```

Comment:
usually for programmatic git commands, I like to be specific to be sure I don't commit files that are not supposed to be committed. If the solution I suggest of storing the yaml file in the inst/ folder is retained, maybe spell out `git add inst/glosario/glossary.yml` (or whatever path/name you choose).

Reply:
Addressed this following your recommendation of `inst/glosario/glossary.yml` in the new PR.